engine/termops.mli

```diff
@@ -227,6 +227,7 @@ val env_rel_context_chop : int -> env -> env * rel_context
 
 (** Set of local names *)
 val vars_of_env: env -> Id.Set.t
+val vars_of_env_mem: env -> (Id.t -> bool)
```

Comment:
I don't think that this function belongs here. It should be moved to `Environ` instead.

Reply:
Do you mean both?

Reply:
Technically both should be there, but for some historical reason the above one was put in that file. I don't know whether you should move the other one in the same PR, though.